projects/controllers/experiments.py

```diff
@@ -66,23 +66,27 @@ def create_experiment(name=None, project_id=None, copy_from=None):
                             project_id=project_id)
     db_session.add(experiment)
     db_session.commit()
+    if copy_from:
+        try:
+            experiment_find = find_by_experiment_id(experiment_id=copy_from)
+            """experiment_find = order_by_dependence(experiment_find)"""
+            for operator in experiment_find['operators']:
+                kwargs = {
+                    "task_id": operator.task_id,
+                    "parameters": operator.parameters,
+                    "dependencies": operator.dependencies,
+                    "position_x": operator.position_x,
+                    "position_y": operator.position_y
+                }
+                create_operator(project_id, experiment.uuid, **kwargs)
+        except Exception:
```

Comment:
@AlendeGeraldo por que essa Exception ocorre? Dá pra substituir por uma Exception mais específica? (é um bom padrão de software substituir Exception genérica por uma específica)

Reply:
@fberanizo  Alterações solicitadas feita

Reply:
@AlendeGeraldo ok... eu vi que você mudou pra "NotFound", mas por que essa exceção pode ocorrer?

Acho estanho usar uma API pra duplicar um experimento, e  receber como resposta: Not found: "não foi possível duplicar". Isso tem cara de erro interno, não de erro NotFound.

Reply:
@AlendeGeraldo se o erro for que o "copyFrom" não existe, por favor siga o padrão do tasks e retorne a exceção "BadRequest('Source experiment does not exist')".

Reply:
@fberanizo Alteração sugerida feita.

Reply:
@AlendeGeraldo então o except deve ser `except NotFound` mesmo.

Porém o `raise BadRequest('Source experiment does not exist')` é mais adequado para retorno do duplicate, já que o parâmetro copyFrom do requestbody  que não está adequado. 

O retorno NotFound é geralmente apenas para os ids na URL, por exemplo: GET `/tasks/ID_QUE_NAO_EXISTE`.

Se o problema é com um valor do requestbody, BadRequest é mais adequado.

Reply:
Talvez a msg não esta adequada para a exceção mas já foi alterada.